netaddr.go

```diff
@@ -29,50 +31,43 @@ const (
 	maxUint16 = 1<<16 - 1
 )
 
+// o4 is the offset to the beginning of the IPv4 address within IP.a.
+const o4 = 12
```

Comment:
Consider `const o4 = len(mapped4Prefix)`. Then you can also `s/len(mapped4Prefix)/o4/` throughout.

Reply:
Done.